serverr/server.js

```diff
@@ -0,0 +1,45 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
+var path = require('path');
+var bcrypt = require('bcryptjs');
+var env = require('dotenv/config');
+var PORT = 8000;
+
+var server = express();
+
+var accessPorts = require('./access/access');
+server.use('/', accessPorts);
+require('./database');
+server.use(express.json());
+server.use(bodyParser.urlencoded({ extended: true }));
+
+
+
+var routes = require('./routes/router');
+server.use('/', routes);
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+server.listen(PORT, () => console.log('Server Is Runing on port ' + PORT));
```

Comment:
```suggestion
server.listen(PORT, () => console.log(`Server Is Runing on port ${PORT}`));
```
es6 interpolation is your friend, use it.